jbpm-wb-kie-server/jbpm-wb-kie-server-backend/src/main/java/org/jbpm/workbench/ks/integration/KieServerDataSetProvider.java

```diff
@@ -414,11 +414,25 @@ protected void handleDataSetGroup(final DataSetDef def,
     protected ColumnType getGroupFunctionColumnType(final DataSetDef def,
                                                     final ColumnGroup columnGroup,
                                                     final GroupFunction groupFunction) {
-        ColumnType type = def.getColumnById(groupFunction.getColumnId()).getColumnType();
+        
+        DataColumnDef column = def.getColumnById(groupFunction.getColumnId());
+        if (column == null) {
+            column =  def.getColumnById(groupFunction.getSourceId());
```

Comment:
```suggestion
            column = def.getColumnById(groupFunction.getSourceId());
```
There's an extra space here. 😛